modules/security-platform-authentication.adoc

```diff
@@ -35,7 +35,7 @@ access policy for groups of users. Application plans allow you to set rate
 limits for API usage and control traffic flow for groups of developers.
 
 For a tutorial on using APIcast v2, the containerized 3scale API Gateway, see
-link:https://support.3scale.net/docs/deployment-options/apicast-openshift[Running APIcast on Red Hat OpenShift]
+link:!https://support.3scale.net/docs/deployment-options/apicast-openshift[Running APIcast on Red Hat OpenShift]
```

Comment:
Not sure about this why, any ideas?

@unleashed
@JStickler
@rh-tokeefe

Thanks!